app/jobs/export_job.rb

```diff
@@ -3,10 +3,17 @@
 class ExportJob < ApplicationJob
   queue_as :default
 
-  def perform(session_uuid, form_short_uuid, start_date, end_date, filename = "export-#{Time.zone.now}.csv")
-    csv_content = Form.find_by_short_uuid(form_short_uuid).to_csv(start_date:, end_date:)
-    ActionCable.server.broadcast(
-      "exports_channel_#{session_uuid}", { csv: csv_content, filename: }
-    )
+  def perform(email, form_short_uuid, start_date, end_date)
+
+    form = Form.find_by_short_uuid(form_short_uuid)
+    record_count = form.response_count
+
+    start_time = Time.now
+    csv_content = form.to_csv(start_date:, end_date:)
+    filename = "export-#{Time.zone.now}.csv"
```

Comment:
Can we use the new filename format here, the one that has the form name embedded?

`touchpoints-form-#{@form.short_uuid}-#{@form.name.parameterize}-responses-#{timestamp_string}.csv`

Reply:
implemented